1209/2357/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: TinyPassword
+owner: Karol Kucza
```

Comment:
The value of the 'owner' field needs to match the name of your org directory, in this case `KarolKucza`. Otherwise, everything else looks good, though it'd be good if you could add a short description of what your device is after the metadata block!